build.sbt

```diff
@@ -258,7 +258,7 @@ lazy val dockerSettings = Def.settings(
         s"curl -L https://git.io/coursier-cli > ${binDir}coursier && chmod +x ${binDir}coursier && coursier install scalafmt --install-dir ${binDir} && rm -rf ${binDir}coursier"
       ),
       Cmd("RUN", "addgroup -g 1002 scala-steward"),
-      Cmd("RUN", "usermod -d /opt/scala-steward -aG 1002 demiourgos728"),
+      Cmd("RUN", "usermod -d /opt/scala-steward -aG 1000,1002 demiourgos728"),
```

Comment:
the group 1000 does not exist in this docker image, I think you need to create it first

Reply:
That's fixed now. Also added docker build testing to the circleci pipeline for PRs.